### DISCUSSION TITLE
Is it planned to support Conformer-CTC models on the Jarvis toolkit?

### DISCUSSION THREAD

**Discussion by Slava715** [NVIDIA-NeMo/NeMo#2487] [Q&A] [answered]
I've tried using Conformer mdel in Jarvis, but I get the following error when loading the model:
![Снимок](https://user-images.githubusercontent.com/68752467/125740123-527e61d9-bca6-4518-9749-38b16ea178d3.PNG)

**Comment by VahidooX** [comment, marked as answer]
Jarvis currently doesn't support Conformer-CTC. We are working to add it for the next releases.

**Reply by Slava715** [reply]
This is very good news, thank you very much :)
